tsconfig.base.json

```diff
@@ -27,6 +27,9 @@
       "@fxa/payments/cart": ["libs/payments/cart/src/index.ts"],
       "@fxa/payments/paypal": ["libs/payments/paypal/src/index.ts"],
       "@fxa/shared/contentful": ["libs/shared/contentful/src/index.ts"],
+      "@fxa/shared/account/account": [
+        "libs/shared/account/account/src/index.ts"
```

Comment:
[nb] Not in love with the `account/account` here

Reply:
Yea, my thought was there would be other account related manager classes, and this one at the data persistence level focuses on the operations for manipulating the Account in our Account system, and there'd be a Devices one, etc. At the moment, it definitely looks..... odd, in isolation.